openhcl/virt_mshv_vtl/src/processor/mshv/arm64.rs

```diff
@@ -220,6 +223,22 @@ impl BackingPrivate for HypervisorBackedArm64 {
     }
 
     fn inspect_extra(_this: &mut UhProcessor<'_, Self>, _resp: &mut inspect::Response<'_>) {}
+
+    fn hv(&self, _vtl: GuestVtl) -> Option<&ProcessorVtlHv> {
```

Comment:
I wonder if we need to return the whole hv object or just the synic.

I'm just thinking that in the ARM case, we might end up having a different hv structure, but we might share the synic structure. Unclear.

Reply:
We currently call msr_read on the hv directly. Presumably a similar function will exist on ARM, even if the list of msrs is different.

Reply:
OK for now. Once we remove support for this stuff from the mshv backend, then this can move into the CVM-specific code.